conversion/conversion.go

```diff
@@ -49,7 +49,44 @@ var (
 	datastreamClient *datastream.Client
 )
 
-func getDatastreamClient(ctx context.Context) *datastream.Client {
+type ConnectionProfile struct {
```

Comment:
nit: this feels like this should reside elsewhere, seems like we are overloading conversion.go again.

Reply:
conversion_types.go? or maybe somewhere in profiles.go? @manitgupta WDYT